### ISSUE TITLE
Help Britain From Above - Not Working since site was re-vamped

### ISSUE THREAD

**Issue by JohnDeZoom** [lovasoa/dezoomify#236]
I've been trying to get full size images from a site that I've successfuly done so from before.
Since the last time I visited the yhave re-vamped the site.
The link to the image I want the full size image for is here:-

https://britainfromabove.org.uk/en/image/EPR000546

Based on what I successfully used previously I've made a small mod and tried this in Dezoomify:-

http://www.britainfromabove.org.uk/sites/all/libraries/aerofilms-images/zoomify512/EPR/000/EPR000546/ImageProperties.xml

But it doesn't work !

Using the Zoomify option I get the message:-

TypeError: response is null 
(http://ophir.alwaysdata.net/dezoomify/zoommanager.js:360)

If I then try the generic dezoomer, it appears to start (TILES LOADING 0% ON SCREEN,) but then I get the message:-

Error: Unable to load tile. Check that your internet connection is working and that you can access this url: http://britainfromabove.org.uk/sites/all/lib/aerofilms-images/zoomify512/EPR/002/EPR000546/ImageProperties.xml (http://ophir.alwaysdata.net/dezoomify/zoommanager.js:178)

The nominal onscreen tile is as follows and is from what I created the url for Dezoomify:-
https://britainfromabove.org.uk/sites/all/libraries/aerofilms-images/zoomify512/EPR/000/EPR000546/TileGroup0/1-0-0.jpg

I can't find any trace of help using Network analysis for the image above or the link at top.

**Comment by dancorpse** [comment]
This site need login to zoom in.  Did this requirement appear before?

**Comment by lovasoa** [comment]
There is a way to make dezoomify work with sites that require login to view zoomable images, but it's a little hidden. In fact, this requires you to fiddle with your browser's console, and uses the server as a proxy to download the tiles, which requires significantly more bandwidth from the server.

**Comment by lovasoa** [comment]
See: https://github.com/lovasoa/dezoomify/issues/27

**Comment by JohnDeZoom** [comment]
oops, just seen your comments lovasoa !
I'll give it a try. since the last time I dezoomified successfully was after that post I guess I followed the instructions (and forgot )  as log in has always been present on that site in order to zoom in.
I'll report back.

**Comment by JohnDeZoom** [comment]
OK I'm using Firefox v. 56.02 (64 bit) ... yes it's not the latest but hey I want to continue using all those lovely old add-ons ! 

OK, the instructions on the #27 issue linked page by lovasoa above  provide me with lots of non-working things !
I guess this is not surprising as approx. 3 years have passed since that work around was written in hich tim Firefox has changed somewhat.

Instructions in issue #27 say:
in firefox, from the site's page, you can do "Ctrl-I" > "Security" > "View Cookies"

after Ctrl I (i.e. Ctrl-shft+I) I get the 'Inspector' window opens but I have no sign of an option 'Security' !!!
Neither 'cookies' anywhere obvious.

So after a googling, on this page: https://www.ghacks.net/2012/11/03/edit-or-remove-cookies-from-firefoxs-developer-toolbar/ 
 I find that I can do the following :

Type Shift-F2' - to display and hide the Developer Toolbar

then in that command line box at bottom I can type:
'cookie list'

and I get a list of the xì4 cookies apparently associated with the page.

I copy these (NOT with right mouse but using the firefox 'Copy' edit command !)


THEN:

To open Console while on Zoomify page

Instructions say: Open the web console inside dezoomify. (In firefox: Ctrl-Maj-K)

For me, Ctrl-Shft-K only works

I then type in (corresponding to instructions):
ZoomManager.cookies ="SESSbf107b399020ce925b024ff380b37230=Q87sE6Rp_t9SEtS8U4ZBtrgQ-mMigX323dHDCSweKEs;_ga=GA1.3.184787457.1526297263;_gid=GA1.3.954577873.1526297263;has_js=1"

and I get the following message appear just above the command line box: 
"scam Warning ... BLAH ... BLAH .....  please type 'allow pasting' (no need to press 'enter' below 

when I do that ALLOW PASTING appears (in upper case' but nothing happens (even if I hit 'enter'
I've also tried re-pasting the ZoomManager.cookies' line above after 'allow pasting' on the same line and nothing

HELP !!!

**Comment by lovasoa** [comment]
It looks like they do not provide the `ImageProperties.xml` file anymore. We would have to add specific support for this site to dezoomify, in order to make it work...

**Comment by JohnDeZoom** [comment]
could it be under another guise (file type) ?
I've searched and searched but can't find anything

**Comment by lovasoa** [comment]
No, it's just a few lines of code to write in dezoomify to handle this particular site, where the meta-information in inside the page itself instead of a separate `.xml` file. But there are tons of new site support requests...

**Comment by JohnDeZoom** [comment]
ok, thanks for putting it on the list I appreciate it.
I look forward to seeing it. Keep up the excellent work !

**Comment by lovasoa** [comment]
Closing as a duplicate site report for britainfromabove.org.uk. The oldest open tracking issue I found for this website is #167; please keep follow-up investigation for this site there.
